validation/lid_driven_cavity_2d/validation_lid_driven_cavity_2d.jl

```diff
@@ -0,0 +1,93 @@
+using TrixiParticles
```

Comment:
Are these added to the tests? I didn't see them.

Reply:
I need #601 for this example